rclpy/rclpy/__init__.py

```diff
@@ -35,8 +35,8 @@ def init(args=None):
     return _rclpy.rclpy_init(args if args is not None else sys.argv)
 
 
-def create_node(node_name):
-    node_handle = _rclpy.rclpy_create_node(node_name)
+def create_node(node_name, namespace=None):
```

Comment:
Since this change is not strictly required, I'd rather not do it in this pr. You're welcome to do a follow up.